scripts/download_wikidata_canadianprovinces.py

```diff
@@ -51,7 +51,7 @@
     for key, name in pairs:
         csv += f"{name.replace(',', ';')},{key.replace(',', ';')}\n"
 
-    with open("wikidata_oceans.csv", "w") as file:
+    with open("wikidata_canadianprovinces.csv", "w") as file:
```

Comment:
We may need to change the file name to just "wikidata_provinces.csv" so that the type becomes "PROVINCES". @lintool please confirm what name to go for here.

Reply:
`wikidata_canadian_provinces.csv`?

Reply:
So the recognized entries will have the type: "**CANADIAN_PROVINCE**"?

Reply:
I think that's fine.